WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Price/ProductPriceSettingsViewController.swift

```diff
@@ -333,17 +333,57 @@ private extension ProductPriceSettingsViewController {
     func configurePrice(cell: UnitInputTableViewCell) {
         let cellViewModel = Product.createRegularPriceViewModel(regularPrice: viewModel.regularPrice, using: CurrencySettings.shared) { [weak self] value in
             self?.viewModel.handleRegularPriceChange(value)
+            self?.setCellPriceAccessibility(price: cell.value, cell: cell)
```

Comment:
Not related to this line.

I found an interesting but very annoying bug that also happens in `develop`. 

1. Activate Price (double-tap).
2. Make changes.
3. Highlight the Sale Price. 
3. Double-tap.

The “Discard Changes” dialog is shown instead of the text field being activated. It's really weird! 😱 

[Here](https://drive.google.com/file/d/13RnnZthNlU7vIzS1eF9jbmAMArnjJ3-i/view?usp=sharing) is a video. We can work on this later since it's not related to this PR.

Reply:
Okay, it looks like this can be fixed by adding this in `UnitInputTableViewCell`:

```swift
final class UnitInputTableViewCell: UITableViewCell {

    override func accessibilityActivate() -> Bool {
        inputTextField.becomeFirstResponder()
        return true
    }

}
```

Reply:
I wasn't able to replicate this issue. 🤔  Are you sure that you don't tap on the left side of the screen like in this issue? https://github.com/woocommerce/woocommerce-ios/issues/2249

Reply:
I was using iPhone 8S. I'll try on iPhone Xs later.

Reply:
I can confirm that it doesn't happen on iPhone Xs. 🤷

Reply:
Sorry, my bad. I can reproduce this on iPhone Xs (13.1). I'm so confusing. 🤪

Reply:
I tried to replicate it, but it doesn't happen to me 🆘 can you try on iOS 13.4?